test/desktop.spec.js

```diff
@@ -76,12 +76,12 @@ describe('Using monkey on desktop', function () {
       });
     });
 
-    it('should not make hyphens in names clickable', function () {
-      $container.find('.nonclickable.special-char .char:contains(-)').length.should.equal(1);
+    it('should not include hyphens in names', function () {
+      $container.find('.char:contains(-)').length.should.equal(0);
```

Comment:
Yes, it was intentional. Names now comes from API without special characters. E.g. Maë-Lily
becomes MAELILY. The API was already working like this, but the specs were pointing to the viking-staging server which was outdated. See this commit: https://github.com/Lostmyname/website/commit/17ed3d5c2f1affba2b8c49ab90f9ffec9c6e50aa